index.html

```diff
@@ -0,0 +1,177 @@
+                <li><a href="https://medium.com/" target="_blank" class="media-link" rel="noopener"><i
+                            class="fab fa-medium-m"></i></a></li>
+                <li><a href="https://github.com/ernestkorir" target="_blank" class="media-link" rel="noopener"><i
+                            class="fab fa-github"></i></a></li>
+                <li><a href="https://angel.co/ernestkorir" target="_blank" class="media-link" rel="noopener"><i
+                            class="fab fa-angellist"></i></a></li>
+            </ul>
+        </div>
+    </section>
+    <section class="work">
+        <div class="card-work-1">
+            <div class="work-image"></div>
+            <div class="left-block">
+                <p class="Tonic">Tonic</p>
+                <div class="frame1">
+                    <p class="canopy">CANOPY</p>
+                    <p class="back-end-dev">Back End Dev</p>
+                    <p class="year">2015</p>
+                </div>
+                <p class="desc">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <div class="tags">
+                    <a class="html">html</a>
+                    <a class="css">css</a>
+                    <a class="javascript">javascript</a>
+                </div>
+                <button class="button">See Project</button>
+            </div>
+
+        </div>
+        <div class="card-work-2">
+            <div class="work-image-2"></div>
+            <div class="left-block">
+                <p class="Tonic">Multi-Post <br> Stories</p>
+                <div class="frame1">
+                    <p class="canopy">CANOPY</p>
+                    <p class="back-end-dev">Back End Dev</p>
+                    <p class="year">2015</p>
+                </div>
+                <p class="desc">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <div class="tags">
+                    <a class="html">html</a>
+                    <a class="css">css</a>
+                    <a class="javascript">javascript</a>
+                </div>
+                <button class="button">See Project</button>
+            </div>
+        </div>
+        <div class="card-work-2">
+            <div class="work-image-3"></div>
+            <div class="left-block">
+                <p class="Tonic">Tonic</p>
+                <div class="frame1">
+                    <p class="canopy">CANOPY</p>
+                    <p class="back-end-dev">Back End Dev</p>
+                    <p class="year">2015</p>
+                </div>
+                <p class="desc">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <div class="tags">
+                    <a class="html">html</a>
+                    <a class="css">css</a>
+                    <a class="javascript">javascript</a>
+                </div>
```

Comment:
- [ ] instead of normal divs, The list of project technologies should be grouped in a  list `ul` and each of them a `li` , kindly fix it